app/views/buyers/applications/_utilization.html.erb

```diff
@@ -20,7 +20,7 @@
         <tr/>
       <% utilization.each do |item| %>
         <tr>
-          <td><b class="u-break-word"><%=item.friendly_name %></b>&nbsp;(<%=item.system_name%>)</td>
+          <td class="u-word-break-all"><b class="u-word-break-word"><%=item.friendly_name %></b>&nbsp;(<%=item.system_name%>)</td>
```

Comment:
```suggestion
          <td class="u-word-break-all"><b><%=item.friendly_name %></b>&nbsp;(<%=item.system_name%>)</td>
```

`b` will inherit the behaviour.

Reply:
Yes, but there is a subtle difference between `break-all` and `break-word` in words including `-`.
With `break-word`applied in `b`, words break in the `-`:
![break-word](https://user-images.githubusercontent.com/13486237/48862537-f08ed480-edc6-11e8-86ed-a183db4be789.png)
Just inheriting from `td`,  words are breaking at any character:
![break-all](https://user-images.githubusercontent.com/13486237/48862581-12885700-edc7-11e8-8e01-cfb4baeee7d9.png)

Which one do you prefer @thomasmaas ?

Reply:
I prefer what you did but I didn’t see that class implemented (but I guess it already was).